models/marts/stg_payments.sql

```diff
@@ -0,0 +1 @@
+select * from raw.stripe.payment
```

Comment:
is this an example of "what not to do" (staging model that doesn't use a source function in the marts folder). I'm not sure if we want to add something like that to this repo since it's intended to act as an exemplar.

Reply:
I actually use this a lot to demo the intro to jinja demo with pivoting stg_payments model. It is definitely lazily implemented as is though. I can drop this, and maybe we can build a better jinja/macro example later on.